WatchFaceAlphaKotlin/app/src/main/java/com/example/android/wearable/alpha/viewmodels/AnalogWatchFaceViewModel.kt

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2020 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.android.wearable.alpha.viewmodels
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.asLiveData
+import com.example.android.wearable.alpha.AnalogWatchFaceService
+import com.example.android.wearable.alpha.data.WatchFaceRepository
+import com.example.android.wearable.alpha.data.db.AnalogWatchFaceEntity
+import com.example.android.wearable.alpha.data.db.WatchFaceArmDimensionsEntity
+import com.example.android.wearable.alpha.data.db.WatchFaceColorStyleEntity
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.MainScope
+import kotlinx.coroutines.launch
+
+/**
+ * View model used for [AnalogWatchFaceService].
+ */
+class AnalogWatchFaceViewModel(private val repository: WatchFaceRepository) {
+
+    // Used to launch coroutines (non-blocking way to insert data).
+    private val scope: CoroutineScope = MainScope()
```

Comment:
Is the ViewModel ever destroyed? 

I'd create a `clear()` method in the viewmodel so we can do `scope.cancel()`